packages/instant-apps-components/src/components/instant-apps-app-guide/instant-apps-app-guide.tsx

```diff
@@ -21,52 +26,72 @@ export class InstantAppsAppGuide {
   @Prop()
```

Comment:
I think it's a good time to start documenting the properties

Example in social share:
https://github.com/Esri/instant-apps-components/blob/master/packages/instant-apps-components/src/components/instant-apps-social-share/instant-apps-social-share.tsx#L104-L106

Once comments are added, you can run `npm run build` and that'll build out the read me doc

Reply:
Done